release.config.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  repositoryUrl: 'https://github.com/PolymeshAssociation/polymesh-rest-api.git',
+  branches: [
+    'master',
+    {
+      name: 'beta',
+      prerelease: true,
+    },
+    {
+      name: 'alpha',
+      prerelease: true,
+    },
```

Comment:
```suggestion
    {
      name: 'develop',
      prerelease: true,
    },
```

Reply:
I think we should switch over to the alpha / beta / master model we have in other repositories so its consistent. I made new branches in this repo and was going to tell people to switch when this got merged.

I am not expecting to use them much, but they could be nice when supporting stuff across chain versions.

Reply:
Will take out beta